### ISSUE TITLE
Switch Case Not Showing "add missing case clauses"

### ISSUE THREAD

**Issue by irajeshh** [dart-lang/sdk#52607]
After upgrading to flutter 3.10.2 the switch case is not showing the `add missing case clauses` fix in menu.

![image](https://github.com/dart-lang/sdk/assets/85046336/78d2b0e0-5207-4e66-a38a-9e94676d55e3)

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.10.2, on macOS 13.3.1 22E772610a darwin-arm64, locale en-IN)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.78.2)
[✓] Connected device (3 available)
    ! Error: Rajesh’s Apple Watch needs to connect to determine its availability. Check the connection between the device and its
      companion iPhone, and the connection between the iPhone and Xcode. Both devices may also need to be restarted and unlocked. (code 1)
[✓] Network resources
```

**Comment by deimantasa** [comment]
Yes, extremely frustrating. It should be tracked [here](https://github.com/dart-lang/sdk/issues/52180).

**Comment by scheglov** [comment]
Works for me.
<img width="734" alt="image" src="https://github.com/dart-lang/sdk/assets/384794/cb85009a-bb17-4816-bcfc-45f521a6f014">

```dart
enum Feature {
  foo, bar
}

void f(Feature feature) {
  switch (feature) {}
}
```

If there are something more specific to your code, don't hesitate to provide additional data.

**Comment by scheglov** [comment]
What is a `Feature`?

I use bleeding edge Dart SDK, but this is not important, the quick assist was there long time.
Try installing something newer, but I doubt the version is the issue here.
https://dart.dev/get-dart/archive

**Comment by irajeshh** [comment]
@scheglov Featrue just an enum I used.
Yes, I know it was there and I used it always, thats' why I created the issue.
Now I've upgraded Flutter SDK to 3.10.4 which uses `Dart 3.0.3` (it's not fixed yet), but from your comment, I came to know that it's fixed in Dart `3.1.0xx`.

**Comment by irajeshh** [comment]
I've changed to beta channel and upgraded to Flutter `3.11.0-0.1.pre` which uses Dart `3.1.0-63.1.beta`. This issue is partially resolved, ie If we have 12 enum values, each time we have to click on switch(foo){//Here
then it will auto add the missing enum only one time, by this way all the 12 will be added manually.
But before it was added in one click.
I hope they will restore it back in upcoming versions.
